LORIS_react.README.md

```diff
@@ -0,0 +1,131 @@
+#LORIS meets ReactJS
+
+##1. Setting up Grunt
+
+In order to run the grunt build step you will have to install the following dependencies
+
+* NodeJS
+* NPM
+
+###1.1 Installing dependencies 
+
+####1.1.1 NodeJS
+
+To check weather or not NodeJS is installed on your machine run the following:
+
+```
+node -v
+```
+
+If not installed use one of the following steps to install for your OS
+
+#####Ubuntu
+
+For this install you will need sudo rights
+
+```
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+sudo apt-get install -y nodejs
+sudo apt-get install -y build-essential
+```
+
+#####CentOS
+
+```
+curl --silent --location https://rpm.nodesource.com/setup_6.x | bash -
+yum -y install nodejs
+yum install gcc-c++ make
+```
+
+#####MacOS
+
+Download the NodeJS package from [here](https://nodejs.org/en/)
+
+####1.1.2 NPM
+
+Once NodeJS is installed use the following to instal NPM
+
+```
+curl -L https://www.npmjs.com/install.sh | sudo sh
+```
+
+###1.2 Installing Grunt
+
+Once NodeJS and NPM are installed you are ready to install Grunt. First in your terminal ```cd``` into the LORIS home directory. 
+Once there run the following command:
+
+```
+npm install --only=dev
+```
+
+Grunt should now be installed 
+
+##2. Compiling JSX
+
+###2.1 Using Grunt to compile all JSX
+
+To compile all JSX files in LORIS use the following command
+
+```
+node_modules/.bin/grunt jsx
```

Comment:
Is there any way to make this command simpler? That's going to be hard for anyone to remember/type.